<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MouseConfig</class>
 <widget class="QWidget" name="MouseConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>123</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_43">
     <property name="text">
      <string>&lt;b&gt;Mouse&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Double click interval:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="doubleClickInterval">
     <property name="suffix">
      <string> ms</string>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Wheel scroll lines:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="wheelScrollLines"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="singleClick">
     <property name="text">
      <string>Single click to activate items</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QCheckBox" name="mouseLeftHanded">
     <property name="text">
      <string>Left handed (Swap left and right mouse buttons)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
